src/main/java/io/github/jamsesso/jsonlogic/utils/DateOperations.java

```diff
@@ -6,11 +6,21 @@
 import java.time.OffsetDateTime;
 
 public class DateOperations {
+    private static final DatatypeFactory dataTypeFactory;
+
+    static {
+        try {
+            dataTypeFactory = DatatypeFactory.newInstance();
+        } catch (DatatypeConfigurationException e) {
+            throw new RuntimeException(e);
```

Comment:
Why are we throwing an unchecked exception? Better to throw a custom exception or worst case just `Exception`